src/TypeSpec.Extension/Emitter.Csharp/test/Unit/modelType.test.ts

```diff
@@ -76,7 +76,6 @@ op test(@body input: Pet): Pet;
                         Kind: "String",
                         IsNullable: false
```

Comment:
Can we have some tests in Emitter to cover  additionalProperties?

Reply:
Sure, I will add some

Reply:
tests added